src/main/java/frc/robot/RobotContainer.java

```diff
@@ -107,16 +107,16 @@ private void initializeClimberCommands() { // TODO change to climber 2020 dur re
     }
   }
 
-  private void initializeShooter2020Commands() {
+  private void initShooter2020() {
     if (RobotConstants.get().hasShooter2020()) {
       shooter = new Shooter2020();
-      // shooter.setDefaultCommand(new ShooterSetCMD(shooter,
-      //   () -> -driverJoystick.getRawAxis(XboxController467.Axes.LeftY.value)
-      // ));
-      operatorShooterFlywheel.whenPressed(new ShooterRunFlywheelCMD(shooter));
-      operatorShooterFlywheel.whenReleased(new ShooterStopFlywheelCMD(shooter));
-      operatorShooterShoot.whenPressed(new ShooterTriggerForwardCMD(shooter));
-      operatorShooterShoot.whenReleased(new ShooterTriggerStopCMD(shooter));
+      shooter.setDefaultCommand(new ShooterSetCMD(shooter,
+        () -> -driverJoystick.getRawAxis(XboxController467.Axes.LeftY.value)
+      ));
+      // operatorShooterFlywheel.whenPressed(new ShooterRunFlywheelCMD(shooter));
+      // operatorShooterFlywheel.whenReleased(new ShooterStopFlywheelCMD(shooter));
+      // operatorShooterShoot.whenPressed(new ShooterTriggerForwardCMD(shooter));
+      // operatorShooterShoot.whenReleased(new ShooterTriggerStopCMD(shooter));
```

Comment:
If you're going to leave a bunch of commented out code, you should add a comment explaining why it's commented out and even better, why/when you should uncomment it and use it.